discretenet/problems/school_bus_scheduling/school_bus_scheduling.py

```diff
@@ -0,0 +1,185 @@
+import numpy.random as random
+from pathlib import Path
+from typing import Union
+import pyomo.environ as pyo
+
+from discretenet.problem import Problem
+from discretenet.generator import Generator
+
+
+class SchoolBusSchedulingProblem(Problem):
+    is_linear = True
+
+    def __init__(self, all_time, schools, routes, time_window, name):
```

Comment:
Docstring describing what these are, and type annotations?

Reply:
Also as with the other classes, you'll need to set `self.schools = schools` etc so that they can be returned by `get_parameters`

Reply:
Updated